app/views/proposals/_archive_link.html.erb

```diff
@@ -1,5 +1,5 @@
 <div class="col-sm-2">
-  <%- unless @approved_data.rows.size < @CLOSED_PROPOSAL_LIMIT %>
-    <%= link_to "View the archive", url_for(:controller => 'proposals', :action => 'archive') %>
-  <%- end %>
+  <% if @approved_data.rows.size > @closed_proposal_limit %>
```

Comment:
we are limiting the results in the controller to  `@closed_proposal_limit` so I think greater than is all we need

previous logic would have showed archive link if there were 10 records, but there would be nothing to more to see in that case